.pre-commit-config.yaml

```
@@ -1,4 +1,24 @@
 repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+    - id: check-added-large-files
+    - id: check-case-conflict
+    - id: check-merge-conflict
+    - id: check-symlinks
+    - id: check-json
+    - id: check-yaml
+    - id: check-toml
+    - id: check-xml
+    - id: debug-statements
+    - id: end-of-file-fixer
+      # exclude generated files
+      exclude: validation/|\.dtd$|\.json$|\.xml$
```

could even make it
```suggestion
      exclude: ^validation/|\.dtd$|\.json$|\.xml$
```
if that's the top-level `validation` directory?

---

Yup! Thanks.